nearcore/src/config.rs

```diff
@@ -487,6 +525,7 @@ impl Default for Config {
             db_migration_snapshot_path: None,
             use_db_migration_snapshot: true,
             enable_rocksdb_statistics: false,
```

Comment:
We should move `enable_rocksdb_statistics` to `store` as well IMO. This option is not present in 1.25.0 so we can still move it around.